packages/app-next/src/App.tsx

```diff
@@ -125,6 +126,7 @@ const app = createApp({
     userSettingsPlugin,
     homePlugin,
     appVisualizerPlugin,
+    azureDevOpsPlugin,
```

Comment:
This shouldn't be needed, if you add the plugin as a dependency it works without this addition as well, right?

Reply:
Removed, thanks 👍